server/events/terraform/terraform_client.go

```diff
@@ -442,11 +444,16 @@ func (c *DefaultClient) RunCommandAsync(ctx models.ProjectCommandContext, path s
 
 		// Asynchronously copy from stdout/err to outCh.
 		go func() {
-			s := bufio.NewScanner(stdout)
-			for s.Scan() {
-				message := s.Text()
-				outCh <- Line{Line: message}
-				c.projectCmdOutputHandler.Send(ctx, message)
+			// Don't stream terraform show output to outCh
+			cmds := strings.Split(tfCmd, " ")
+			// OverrideTF used for testing with non terraform commands.
+			if c.overrideTF != "" || isValidCommand(cmds[1]) {
+				s := bufio.NewScanner(stdout)
+				for s.Scan() {
+					message := s.Text()
+					outCh <- Line{Line: message}
```

Comment:
This needs to always run. `outCh` is used to display the terraform command output on the PR. `isValidCommand` is only relevant for `c.projectCmdOutputHandler.Send(ctx, message)`

Reply:
True! Thanks for the catch :)